src/Corporation/ui/modals/SellDivisionModal.tsx

```diff
@@ -59,8 +58,8 @@ export function SellDivisionModal(props: IProps): React.ReactElement {
         <Typography>
           Profit: <Money money={(divisionToSell.lastCycleRevenue - divisionToSell.lastCycleExpenses) / 10} /> / sec{" "}
         </Typography>
-        <Typography>Cities:{getRecordKeys(divisionToSell.offices).length}</Typography>
-        <Typography>Warehouses:{getRecordKeys(divisionToSell.warehouses).length}</Typography>
+        <Typography>Cities: {getRecordKeys(divisionToSell.offices).length}</Typography>
+        <Typography>Warehouses: {getRecordKeys(divisionToSell.warehouses).length}</Typography>
```

Comment:
This section might look better as a `<StatsTable>`, which automatically aligns the right column.